### ISSUE TITLE
how to resend a confirmation email ?

### ISSUE THREAD

**Issue by m2omou** [lynndylanhurley/devise_token_auth#390]
I was wondering, what if the user didn't receive the confirmation email, is there any way I can resend it ?

Something like "if you didn't receive the confirmation email please click here"

I guess there is a route for this ?

Thanks.

**Comment by ACPK** [comment]
+ 1. Would be nice if this was automatically (but only once) sent if a user gets the error message above.

**Comment by m2omou** [comment]
@ACPK  Yes that would be nice, most website have something similar. Any update ?

**Comment by m2omou** [comment]
any news on that one ? :)

**Comment by v3rron** [comment]
will probably need to write this part myself :|

**Comment by m2omou** [comment]
+1

**Comment by dchersey** [comment]
+1 -- anybody have code for this? Seems like you have to override ConfirmationsController#new and call send_confirmation_instructions from the model concern.  

Shouldn't be too hard but why reinvent the wheel?

**Comment by v3rron** [comment]
Thanks to mkhatib providing a solution on the related ng-token-auth issue: 
https://github.com/lynndylanhurley/ng-token-auth/issues/79#issuecomment-73386934

**Comment by BrahimDahmani** [comment]
+1

**Comment by Ruff9** [comment]
+1

**Comment by alfredoreduarte** [comment]
I just ran into the same issue, but where should I put the class extension?

**Comment by ethagnawl** [comment]
@zachfeldman Would a PR containing [the solution mentioned above](https://github.com/lynndylanhurley/devise_token_auth/issues/390#issuecomment-203063396) (or similar) be considered? It seems like this is a pretty common use case.

**Comment by zachfeldman** [comment]
@ethagnawl I would certainly be open to it!

**Comment by lynndylanhurley** [comment]
@ethagnawl please submit a PR for this if you have time.
